build_tarballs.jl

```diff
@@ -44,8 +42,8 @@ products(prefix) = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    "https://github.com/bicycle1885/ZlibBuilder/releases/download/v1.0.2/build_Zlib.v1.2.11.jl",
-    "https://github.com/SimonDanisch/LibpngBuilder/releases/download/v1.0.1/build_libpng.v1.6.31.jl",
+    "https://github.com/bicycle1885/ZlibBuilder/releases/download/v1.0.4/build_Zlib.v1.2.11.jl",
+    "https://github.com/SimonDanisch/LibpngBuilder/releases/download/v1.0.2/build_libpng.v1.6.31.jl",
     "https://github.com/SimonDanisch/LibJPEGBuilder/releases/download/v9b/build_libjpeg.v9.0.0-b.jl",
     "https://github.com/SimonDanisch/LibTIFFBuilder/releases/download/v1.0.0/build_libtiff.v4.0.9.jl"
```

Comment:
This PR is draft because this dependency needs to be updated after https://github.com/SimonDanisch/LibTIFFBuilder/pull/1 is merged